.pre-commit-config.yaml

```diff
@@ -67,7 +67,7 @@ repos:
     hooks:
     - id: codespell
       name: Check spelling (codespell)
-      args: ["--ignore-words-list", "additionals,curren,fo,ist,ket,superseeding,te,ths"]
+      args: ["--ignore-words-list", "additionals,curren,fo,ist,ket,superseeding,te,ths,searchIn"]
```

Comment:
I tested it myself. 2 suggestions here:
1. keep the list sorted.
2. the word needs to be all lowercase for it to work.
```suggestion
      args: ["--ignore-words-list", "additionals,curren,fo,ist,ket,searchin,superseeding,te,ths"]
```

Reply:
Thanks for the help here